### PR TITLE
[Several] Fix typo (II)

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

This is part of #10180. Cherry-picking little by little until #10180 is manageable.

(No parameters or project parameters are modified here)

**🆕 Changelog**

- [Typo fix in DropletDynamicsApp](https://github.com/KratosMultiphysics/Kratos/commit/97bc7dbcc71828b01a0e1ebbd5923281209b40d0)
- [Typo fix in ExaquteApp](https://github.com/KratosMultiphysics/Kratos/commit/8880b6fde4f6950af9070cbe9fba3531469f1372)
- [Typo fix in DEMApp](https://github.com/KratosMultiphysics/Kratos/commit/b55f5fa88ff9fa1ae66814b9c20b45ca3ba05ec0)
- [Typo fix in ConventionDiffusionApp](https://github.com/KratosMultiphysics/Kratos/commit/654bc91070a77602a5b1afd78a93bccfb2b0f7f4)
- [Typo fix in SolidApp](https://github.com/KratosMultiphysics/Kratos/commit/2e8b4e2384a6ee73460d6cdef7f64f6103b2240f)
- [Typo fix in TrilinosApp](https://github.com/KratosMultiphysics/Kratos/commit/6d2c2ae591f22313b7dbb77916d9ab48a66b610e)
- [Typo fix in PFEMSolidApp](https://github.com/KratosMultiphysics/Kratos/commit/611bc0c12b21bf4e99a963390a703f8337bf87b1)

### CODE REVIEW

File: applications/TrilinosApplication/custom_python/trilinos_pointer_wrapper.h [hunk 1]
```diff
@@ -163,7 +163,7 @@ class AuxiliaryMatrixWrapper
     ///@}
 }; // Class AuxiliaryMatrixWrapper
 
-/// Auxiliarty Trilinos vector pointer wrapper class
```
Comment (RiccardoRossi) [inline applications/TrilinosApplication/custom_python/trilinos_pointer_wrapper.h]: your replace is wrong here

File: applications/TrilinosApplication/custom_utilities/trilinos_cutting_app.h [hunk 2]
```diff
@@ -333,7 +333,7 @@ class TrilinosCuttingApplication
 
         for (int index=0; index!=nnodes; ++index)  //now we have to save the nodes!.
         {
-             //KRATOS_WATCH(auxiliar);
```
Comment (RiccardoRossi) [inline applications/TrilinosApplication/custom_utilities/trilinos_cutting_app.h:336]: this you can remove

File: applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h [hunk 3]
```diff
@@ -268,7 +268,7 @@ class TrilinosCuttingIsosurfaceApplication
 
         for (int index=0; index!=nnodes; ++index)  //now we have to save the nodes!.
         {
-            // KRATOS_WATCH(auxiliar);
```
Comment (RiccardoRossi) [inline applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h:271]: same

File: applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp [hunk 4]
```diff
@@ -174,9 +174,9 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("GetInterfaceResidualSize", &TrilinosPartitionedFSIUtilitiesArray3DType::GetInterfaceResidualSize)
         .def("SetUpInterfaceVector", [](TrilinosPartitionedFSIUtilitiesArray3DType& self, ModelPart& rModelPart){
             return AuxiliaryVectorWrapper(self.SetUpInterfaceVector(rModelPart));})
-        .def("UpdateInterfaceValues", &AuxiliarUpdateInterfaceValues<array_1d<double,3>,3>)
```
Comment (RiccardoRossi) [inline applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp:177]: this changes the API ...

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
you are changing the API once again...

File: applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp (hunk 4 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp:177]: AuxiliarUpdateInterfaceValues It is defined in the same file, cannot be accessed from other side, and is exported to python with a different name, doesn't break the API

Comment (loumalouomega) [reply applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp:177]: See the whole file, applications/TrilinosApplication/custom_python/add_custom_utilities_to_python.cpp, the definition is not accessible outside this file and exported with different name

File: applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h (hunk 3 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h:271]: Done

File: applications/TrilinosApplication/custom_utilities/trilinos_cutting_app.h (hunk 2 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_utilities/trilinos_cutting_app.h:336]: Done

File: applications/TrilinosApplication/custom_python/trilinos_pointer_wrapper.h (hunk 1 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_python/trilinos_pointer_wrapper.h]: Done

(Comments better in the changed part to track changes)

File: applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h (hunk 3 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h:271]: BTW, these two file the code is super duplicated

**Comment by loumalouomega** [comment]
@RiccardoRossi I attended your suggestions

**Comment by rubenzorrilla** [comment]
@loumalouomega please note the @KratosMultiphysics/technical-committee 's observations in [here](https://github.com/KratosMultiphysics/Kratos/pull/10180#pullrequestreview-1121593809). We still think that the changes are not worth, even though they are separated into smaller PRs. We also note that, even though it's archaic, `auxiliar` is a correct form. 

Having said this, we suggest avoiding these changes.

**Comment by loumalouomega** [comment]
> @loumalouomega please note the @KratosMultiphysics/technical-committee 's observations in [here](https://github.com/KratosMultiphysics/Kratos/pull/10180#pullrequestreview-1121593809). We still think that the changes are not worth, even though they are separated into smaller PRs. We also note that, even though it's archaic, `auxiliar` is a correct form.

It is not archaic: https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=auxiliar or https://www.dictionary.com/misspelling?term=auxiliar, is not even in the dictionary (in Dictionary.com the archaisms are included, P.E: https://www.dictionary.com/browse/bawcock)

**Comment by rubenzorrilla** [comment]
> > @loumalouomega please note the @KratosMultiphysics/technical-committee 's observations in [here](https://github.com/KratosMultiphysics/Kratos/pull/10180#pullrequestreview-1121593809). We still think that the changes are not worth, even though they are separated into smaller PRs. We also note that, even though it's archaic, `auxiliar` is a correct form.
> 
> It is not archaic: https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=auxiliar or https://www.dictionary.com/misspelling?term=auxiliar, is not even in the dictionary (in Dictionary.com the archaisms are included, P.E: https://www.dictionary.com/browse/bawcock)

See [here](https://www.websters1913.com/words/Auxiliar). In any case, the opinion of the @KratosMultiphysics/technical-committee is the very same.

**Pull request closed without merging**